### PR TITLE
Fix Heron warnings

### LINKED ISSUE #6716: Bug in Heron USV simulations
**Describe the Bug**
Start the `swarm.wbt` or `ocean.wbt `worlds and the simulation will not start (only black screen).
The following warnings appear:
```
ERROR: Error downloading EXTERNPROTO 'OilBarrel': Cannot download 'https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/obstacles/protos/OilBarrel.proto', error code: 203: Error transferring https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/obstacles/protos/OilBarrel.proto - server replied: 
ERROR: Error downloading EXTERNPROTO 'TexturedBackground': Cannot download 'https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/backgrounds/protos/TexturedBackground.proto', error code: 203: Error transferring https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/backgrounds/protos/TexturedBackground.proto - server replied: 
ERROR: Error downloading EXTERNPROTO 'TexturedBackgroundLight': Cannot download 'https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/backgrounds/protos/TexturedBackgroundLight.proto', error code: 203: Error transferring https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/backgrounds/protos/TexturedBackgroundLight.proto - server replied: 
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':14:1: error: Skipped unknown 'TexturedBackground' node or PROTO.
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':16:1: error: Skipped unknown 'TexturedBackgroundLight' node or PROTO.
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':40:1: error: Skipped unknown 'OilBarrel' node or PROTO.
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':56:1: error: Skipped unknown 'OilBarrel' node or PROTO.
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':73:1: error: Skipped unknown 'OilBarrel' node or PROTO.
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':90:1: error: Skipped unknown 'OilBarrel' node or PROTO.
WARNING: Heron "heron":  Skipped node: Cannot insert Transform node in 'boundingObject' field of Robot node in bounding object.
WARNING: Heron "heron(3)":  Skipped node: Cannot insert Transform node in 'boundingObject' field of Robot node in bounding object.
WARNING: Heron "heron(1)":  Skipped node: Cannot insert Transform node in 'boundingObject' field of Robot node in bounding object.
WARNING: Heron "heron(2)":  Skipped node: Cannot insert Transform node in 'boundingObject' field of Robot node in bounding object.
WARNING: Heron "heron": Mass is invalid because 'boundingObject' is not defined. Using default mass properties: mass = 1, inertia matrix = identity
WARNING: Heron "heron(3)": Mass is invalid because 'boundingObject' is not defined. Using default mass properties: mass = 1, inertia matrix = identity
WARNING: Heron "heron(1)": Mass is invalid because 'boundingObject' is not defined. Using default mass properties: mass = 1, inertia matrix = identity
WARNING: Heron "heron(2)": Mass is invalid because 'boundingObject' is not defined. Using default mass properties: mass = 1, inertia matrix = identity
WARNING: Heron "heron" > DEF HERON Shape > Mesh "": Too many triangles (23409) in mesh: unable to cast shadows, please reduce the number of triangles below 21845 or set Shape.castShadows to FALSE to disable this warning.
WARNING: Heron "heron(3)" > DEF HERON Shape > Mesh "": Too many triangles (23409) in mesh: unable to cast shadows, please reduce the number of triangles below 21845 or set Shape.castShadows to FALSE to disable this warning.
WARNING: Heron "heron(1)" > DEF HERON Shape > Mesh "": Too many triangles (23409) in mesh: unable to cast shadows, please reduce the number of triangles below 21845 or set Shape.castShadows to FALSE to disable this warning.
WARNING: Heron "heron(2)" > DEF HERON Shape > Mesh "": Too many triangles (23409) in mesh: unable to cast shadows, please reduce the number of triangles below 21845 or set Shape.castShadows to FALSE to disable this warning.
```

**Steps to Reproduce**
1. Start the one of the sample simulations: `swarm.wbt` or `ocean.wbt `

**Expected behavior**
Simulation runs smoothly, no warnings and errors.

**System**
 - Operating System: Ubuntu 22.04
 - Graphics Card: [RENOIR (renoir, LLVM 15.0.7, DRM 3.57, 6.8.0-49-generic)

### PR DESCRIPTION
**Pull request by MNikoliCC**
**Description**
The `EXTERNPROTO` import paths for certain PROTO files were incorrectly referencing raw GitHub URLs, such as `https://raw.githubusercontent.com/cyberbotics/webots/R2025a/`.... This has been fixed by replacing them with the correct Webots-specific protocol `webots://`.

**Related Issues**
Fixes #6716

**Tasks**
Add the list of tasks of this PR.
  - [x] Fix bugs

### CODE REVIEW

**Comment by MNikoliCC** [comment]
This is the new warning that produces the Heron USV robot to fall into the sea, because there is no bounding object:
```
WARNING: Heron "heron":  Skipped node: Cannot insert Transform node in 'boundingObject' field of Robot node in bounding object.
WARNING: Heron "heron": Mass is invalid because 'boundingObject' is not defined. Using default mass properties: mass = 1, inertia matrix = identity
```
A workaround would be to make a simple bounding object from multiple Pose nodes instead of loading the whole mesh and scaling it down. This would reduce the complexity of the simulation as well.

**Comment by MNikoliCC** [comment]
After adding the boundingObject, the simulation is working as expected.

File: docs/reference/changelog-r2025.md [hunk 1]
```diff
@@ -49,3 +49,4 @@ Released on December **th, 2023.
     - Removed the old `wb_supervisor_field_import_sf_node` and `wb_supervisor_field_import_mf_node` functions from the list of editor autocomplete suggestions ([#6701](https://github.com/cyberbotics/webots/pull/6701)).
     - Fixed handling of remote assets from unofficial sources ([#6585](https://github.com/cyberbotics/webots/pull/6585)).
     - Fixed the QtToolTip bug, where the text in the text box is not being displayed ([#6711](https://github.com/cyberbotics/webots/pull/6711)).
+    - Fixed Heron USV warnings regarding the importing of PROTO files and the boundingObject description ([#6717](https://github.com/cyberbotics/webots/pull/6717)).
```
Comment (lukicdarkoo) [inline docs/reference/changelog-r2025.md:52]: This is not needed as the model wasn't present in the previous Webots version

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]
Looks good, just remove the changelog

**Review by lukicdarkoo — APPROVED** [review]
Thank you

**Comment by MNikoliCC** [comment]
Whit this change (decimating the mesh), the last warning is fixed:
`WARNING: Heron "heron" > DEF HERON Shape > Mesh "": Too many triangles (23409) in mesh: unable to cast shadows, please reduce the number of triangles below 21845 or set Shape.castShadows to FALSE to disable this warning.`

**Review by lukicdarkoo — APPROVED** [review]

**Pull request merged**